.github/workflows/code_sample_checker.yml

```
@@ -0,0 +1,34 @@
+name: Code sample checker
+on: pull_request
+
+jobs:
+    test-code-samples:
+        name: check-code-samples
```

```suggestion
        name: Check code samples
```
Let's make it more human-readable :smile: